Hotsapi.Parser.Test/ReplayParse.tests.cs

```diff
@@ -0,0 +1,82 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Text;
+using System.IO;
+
+namespace Hotsapi.Parser.Test
+{
+    [TestClass]
+    public class ProgramTest
+    {
+        private string fixturePath = "";
+
+        [TestInitialize]
+        public void Init()
+        {
+            fixturePath = Path.Combine(new DirectoryInfo(Environment.CurrentDirectory).Parent.Parent.Parent.FullName, "Fixtures");
+        }
+
+        private void RunTest(string name)
```

Comment:
Can we scan for files in Fixtures dir and execute test on all of them, so that when we want to add a new test case we only need to drop .StormReplay and expected .json there? Future contributors might forget to also add a test method.

Reply:
The idea behind adding each test manually is that I wanted to get individual test feedback in the editor. If one of the test scenarios failed I wanted to know which one.

 I think this would be trivial in something like Node.js/mocha:

```
for (const name of names {
  it(name, () => {
    // test code
  })
}
```

But I'm not sure how this would be done in C#.

Reply:
It seems there are data driven unit test support: https://stackoverflow.com/a/17805540/282328

now only need to figure out how to use a file list instead of .csv as data source

Reply:
One user suggested a workaround: https://stackoverflow.com/a/43282116/282328

add a command that generates a file with test file list during the build

Reply:
Another proposed solution was to use test templates: https://stackoverflow.com/a/8073735/282328

but I think it is more complicated, but maybe I'm wrong and .tt template will just work without any additional config

Reply:
Okay, I've figured something out.

Reply:
Nice! Short and more readable than my examples.